src/components/ProductTours/LearnerCreditTour.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ProductTour } from '@edx/paragon';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Cookies from 'universal-cookie';
+
+import { ROUTE_NAMES } from '../EnterpriseApp/constants';
+import { SETTINGS_TABS_VALUES } from '../settings/data/constants';
+import { features } from '../../config';
+import {
+  LEARNER_CREDIT_COOKIE_NAME,
+  TOUR_TARGETS,
+} from './constants';
+import { SubsidyRequestsContext } from '../subsidy-requests';
+
+const cookies = new Cookies();
+
+const LearnerCreditTour = ( enterpriseSlug,  tourEnabled , setTourEnabled) => {
+  const history = useHistory();
+  const disableTour = () => {
+    cookies.set(
+      LEARNER_CREDIT_COOKIE_NAME,
+      true,
+      { sameSite: 'strict' },
+    );
+  };
+
+  const handleDismissTour = () => {
+    disableTour();
+    setTourEnabled(false);
+  };
+
+  const handleGoToLearnerCreditPage = () => {
+    disableTour();
+    history.push({ pathname: `/${enterpriseSlug}/admin/learner-credit` });
+  };
+
+  const tour = {
+    tourId: 'LearnerCreditTour',
+    endButtonText: 'Continue To Learner Credit Page',
+    dismissButtonText: 'Dismiss',
+    enabled: tourEnabled,
+    onDismiss: handleDismissTour,
+    onEnd: handleGoToLearnerCreditPage,
+    checkpoints: [
+      {
+        placement: 'right',
+        body: "We've recently added a new feature that enables admins to review offer spend and participation. "
+          + 'Continue to the learner credit page to learn more.',
```

Comment:
nit: Should this say "Learner Credit Management page" if that's the full page name? Lol, but then it's inconsistent with the `endButtonText` (no "Management" there, nor could the button fit it).